packages/docs/PropTables/HeaderPropTable.tsx

```diff
@@ -11,34 +11,59 @@ const headerProps: Prop[] = [
   },
   {
     name: 'actions',
-    types: 'Array<ActionButtonProps | ActionDropdownProps>',
+    types: (
+      <>
+        {'Array<'}
+        <NextLink href={{ hash: 'action-button-prop-table', query: { props: 'action-button' } }}>
+          ActionButton
+        </NextLink>
+        {' | '}
+        <NextLink
+          href={{ hash: 'action-dropdown-prop-table', query: { props: 'action-dropdown' } }}
+        >
+          ActionDropdown
+        </NextLink>
+        {'>'}
+      </>
+    ),
     description: 'An array of actions to display in the header. Can include buttons or dropdowns.',
   },
   {
     name: 'backLink',
-    types: 'BackLinkProps',
+    types: (
+      <NextLink href={{ hash: 'backlink-prop-table', query: { props: 'back-link' } }}>
+        BackLink
+      </NextLink>
+    ),
     description: 'Configuration for a back link displayed at the top left of the header.',
   },
   {
     name: 'badge',
-    types: 'BadgeProps',
+    types: (
+      <NextLink href="/badge#props" key="badge-type">
+        BadgeProps
+      </NextLink>
+    ),
     description: 'Props to display a badge next to the title.',
   },
   {
     name: 'description',
-    types: 'string | ReactNode',
+    types: [
+      'string',
+      <NextLink
+        href={{ pathname: '/typography', query: { implementation: 'text', props: 'text' } }}
+        key="text-type"
+      >
+        Text
+      </NextLink>,
+    ],
     description: 'A description to display under the title.',
   },
   {
     name: 'icon',
     types: 'ReactNode',
     description: 'An icon to display to the left of the title.',
   },
-  {
-    name: 'children',
-    types: 'ReactNode',
-    description: 'Any additional content to render inside the header.',
-  },
```

Comment:
`children` is not actually passed anywhere in the component so I think we'll need to remove the `PropsWithChildren` on the type.